browser-interface/static/preview.html

```diff
@@ -537,6 +537,23 @@
 
     <script>
       globalThis.preview = true
+
+      const queryParameters = new URLSearchParams(document.location.search)
+      if (queryParameters.has('PIPE_CONSOLE_TO_PARENT')) {
+        function pipe(type, originalFunctionCallback) {
+          return (...args) => {
+            const targetWindow = window.parent || window
+            targetWindow && targetWindow.postMessage({ type, payload: args }, '*')
+            originalFunctionCallback(...args)
+          }
+        }
+
+        console.log('Pipe the console log and error enabled.')
+
+        globalThis.originalConsole = { log: globalThis.console.log, error: globalThis.console.error }
+        globalThis.console.log = pipe('log', globalThis.originalConsole.log)
```

Comment:
I think should only pipe the scene's console from the dev adapter. Wdyt?

Reply:
I think we'd miss a lot of info in the editor if we only handle the scene logs.

Reply:
Isn't that the only thing that matters? There are toons of useless logs there, if we'd like to debug we can open vscode's own console to see the logs

Reply:
It **should be** the only thing that matters. But this enables the possibility to filter other things that can matter. I agree the Dev-Adapter approach is the ideal one